config/autoload/global.php

```diff
@@ -57,12 +57,21 @@
         'max_photos_page' => 20,
         'dir_mode' => 0777, // rwx by default
         'small_thumb_size' => array(
-            'width' => 180,
-            'height' => 120
+            'width' => 825,
+            'height' => 550
```

Comment:
Ja dat is voor nu goed genoeg, we moeten aan het eind nog even kijken wat de optimale resoluties zijn. Het maakt voor de implementatie verder niet zo veel uit.